masking/mask_terms.py

```diff
@@ -127,38 +215,33 @@ def main():
             line = line.rstrip('\n')
 
         if args.unmask:
-            tokens = line.split('\t')
-            if len(tokens) != 3:
-                raise Exception('Need three fields (output, orig source, and masked source)!')
-
-            # output, orig_source, masked_source
-            unmasked = masker.unmask(*tokens)
-            if args.json:
-                jobj['unmasked'] = unmasked
-                print(json.dumps(jobj), flush=True)
-            else:
-                print(unmasked, flush=True)
+            if not args.json:
+                raise Exception('Unmasking requires json format')
+            
+            output = jobj['masked_output']
```

Comment:
Can we do `masked_text` for the field name for consistency here? I haven't written the JSON spec but I've been working with fields like `raw_text`, `translated_text`, and so on.

Reply:
This should be whatever the indexed masked MT output is. Maybe 'masked_translation'?